ADOPTERSandVENDORS.md

```diff
@@ -13,3 +13,12 @@ A non-exhaustive, alphabetized list of organizations that have adopted OpenFeatu
 | [Virtru](https://www.virtru.com)                  |            |       |
 
 _Sorted alphabetically by name_
+
+# Vendors
+
+A non-exhaustive, alphabetized list of organizations that have offer an OpenFeature Provider.
```

Comment:
Can we just link to the ecosystem page?

Reply:
I would like to keep this separate, to clarify which logos have been cleared to showcase on the website